code/__DEFINES/preferences.dm

```diff
@@ -131,6 +131,15 @@ GLOBAL_LIST_INIT(helmet_styles, list(
 #define RANDOMIZE_SPECIES (1<<0)
 #define RANDOMIZE_NAME (1<<1)
 
+// values for /datum/preference/infotab_category
+/// This belongs to additional name setting (clown name, cyborg name, etc)
+#define PREFERENCE_INFOTAB_NAMES "names"
+/// This belongs to additional game setting (PDA color or theme, securty department preference, etc)
+#define PREFERENCE_INFOTAB_GAMES "games"
```

Comment:
No no no no no, use PREFERENCE_CATEGORY defines. They already exist and do exactly what you are recreating. Hardcode specific categories to tabs in TGUI like all other categories already do.